src/users/users.controller.ts

```diff
@@ -307,6 +313,42 @@ export class UsersController {
     return this.usersService.findOneAndDeleteUserSettings(id);
   }
 
+  @UseInterceptors(DefaultUserSettingsInterceptor)
+  @UseGuards(
+    class ByPassAuthenticatedPoliciesGuard
+      extends PoliciesGuard
+      implements CanActivate
+    {
+      async canActivate(): Promise<boolean> {
+        return Promise.resolve(true);
+      }
+    },
+  )
```

Comment:
If the endpoint should open to unauthenticated users, we can just use PoliciesGuard or remove UseGuards at all I suppose. Since we are not checking policies here